free-2.md

```diff
@@ -11,41 +11,88 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+There is a need to compile and link C/C++ files by the software development community.
+Currently, the common way of compiling and linking C/C++ code is through a DSL called
+make. However, make is very very confusing to use and doesnt seem to have much fluency
```

Comment:
This sounds like a great project. I was just having troubles with 70 students this past week learning makefiles because both the students and I were confused on parts of make. With a project like this, it would also be interesting to look at it as a learning tool. In the end, that ends up being a design decision: Are you building this so that people can compile and link faster or so that people can more intuitively see the linking?